libs/utils/source/ftrack_utils/extensions/overrides.py

```diff
@@ -24,7 +24,9 @@ def set_overrides(current_extensions, new_extensions):
             ):
                 existing_extension = discovered_extension
                 break
-            elif not new_extension['extension_type'].endswith('_config'):
+            elif not new_extension['extension_type'].endswith(
+                '_config'
+            ) and not new_extension['extension_type'].startswith('js_'):
```

Comment:
Should we align them and do _js at the end? than in a previous line we set up all the non pythonic extensions?